src/utils/CAPTCHASolver.ts

```
@@ -0,0 +1,573 @@
+        console.log('🛠️ [CAPTCHASolver] Starting image preprocessing...')
+
+        try {
+            const outputPath = path.join(
+                this.tempDir,
+                `captcha_preprocessed_${Date.now()}.png`,
+            )
+            console.log(`📁 [CAPTCHASolver] Output path: ${outputPath}`)
+
+            let ffmpegArgs = ['-i', inputPath]
+
+            // Apply preprocessing steps
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'ffmpegArgs' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)